packages/contracts/contracts/GenArt721CoreV3.sol

```diff
@@ -1479,6 +1527,18 @@ contract GenArt721CoreV3 is
         scriptCount = project.scriptCount;
     }
 
+    /**
+     * @notice Returns the compressed form of a string in bytes using solady LibZip's flz compress algorithm.
```

Comment:
WDYT about noting that the bytes returned from this function are intended to be used as inputs to the add script functions

Reply:
Good call - updated the notice here to include intended use